src/data/finsupp/pointwise.lean

```diff
@@ -48,17 +48,21 @@ begin
 end
 end
 
-instance [semiring β] : semigroup (α →₀ β) :=
+instance [semigroup_with_zero β] : semigroup (α →₀ β) :=
```

Comment:
Can the conclusion be promoted to a `semigroup_with_zero (α →₀ β)`?

Reply:
Good point. Now done in https://github.com/leanprover-community/mathlib/pull/6786/commits/6791a868c55c8eff2afa63d81b1cfb775779b3de